package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opensearch-dashboards-test/opensearch-dashboards-test-library",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
we may start to have version management for this package. when/how to bump up.

Reply:
This is not part of code change, my bad I don't merge first. it is gone now.

Reply:
i guess you mean rebase :)

btw, it is not in scope of PR, but we need version strategy for this package so we could scale while multiple people work on it

Reply:
with the package.json in functional-test pointed to test-library git repo, we don't need version update for every test-library change. For my local development, I forked this test-library, and in my functional-test dependency it points to my forked repo.